### PR TITLE
feat(plugin): support custom appearance for local dataset

### PR DESCRIPTION
**Pull request by airslice**
## Overview

This PR supports customize project name / logo / theme color.

## How to

- Set custom logo & project name in sidebar's property.
- Set theme color in scene property -> publish theme (custom) -> select color. This will effect plateauview plugin and builtin plugin at the same time.

## Test
- Test project [here](https://test.reearth.dev/published.html?alias=hjheghjhcg).

## Screen shot

test reearth dev_published html_alias=hjheghjhcg

### CODE REVIEW

File: plugin/web/extensions/sidebar/core/components/content/Share/hooks.ts [hunk 1]
```diff
@@ -135,15 +135,19 @@ function generatePrintView(payload?: string) {
   };
   const localizedDate = currentDate.toLocaleDateString("ja", options);
 
+  const themeColor = getComputedStyle(document.documentElement).getPropertyValue(
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/content/Share/hooks.ts:138]: Can you use getComputedStyle inside a sandboxed iframe????

File: plugin/web/extensions/sidebar/core/components/mobile/index.tsx [hunk 2]
```diff
@@ -91,9 +93,19 @@ const MobileSidebar: React.FC<Props> = ({ className }) => {
...
   return (
     <Wrapper className={className}>
-      <PlateauIcon icon="plateauLogo" size={114} wide />
+      {!useCustomProjectHeader && <PlateauIcon icon="plateauLogo" size={114} wide />}
+      {useCustomProjectHeader && (
+        <CustomProjectHeader>
+          {customLogo && <CustomLogo src={customLogo} />}
+          {customProjectName && <CustomProjectName>{customProjectName}</CustomProjectName>}
+        </CustomProjectHeader>
+      )}
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/mobile/index.tsx:108]: Can you do a ternary operator instead? So fall-backing to the plateauLogo

File: plugin/web/extensions/sidebar/core/components/content/Share/hooks.ts (hunk 1 above)
Comment (airslice) [reply plugin/web/extensions/sidebar/core/components/content/Share/hooks.ts:138]: I think so. It does not require parent's window, right?

Comment (KaWaite) [reply plugin/web/extensions/sidebar/core/components/content/Share/hooks.ts:138]: Oh my mistake I think

**Review by KaWaite — APPROVED** [review]

**Pull request merged**